test/tests/treeview-2a.js

```diff
@@ -0,0 +1,723 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const assertAttributeValues = require('../util/assertAttributeValues');
+const assertAriaLabelledby = require('../util/assertAriaLabelledby');
+const assertRovingTabindex = require('../util/assertRovingTabindex');
+
+const exampleFile = 'treeview/treeview-2/treeview-2a.html';
+
+const ex = {
+  treeSelector: '#ex1 [role="tree"]',
+  treeitemSelector: '#ex1 [role="treeitem"]',
+  groupSelector: '#ex1 [role="group"]',
+  folderSelector: '#ex1 [role="treeitem"][aria-expanded]',
+  topLevelFolderSelector: '#ex1 [role="tree"] > [role="treeitem"]',
+  nextLevelFolderSelector: '[role="group"] > [role="treeitem"][aria-expanded]',
+  linkSelector: '#ex1 a[role="treeitem"]'
+};
+
+const openAllFolders = async function (t) {
+  const closedFoldersSelector = ex.treeitemSelector + '[aria-expanded="false"]';
+  let closedFolders = await t.context.session.findElements(By.css(closedFoldersSelector));
+
+  // Going through all closed folder elements in dom order will open parent
+  // folders first, therefore all child folders will be visible before clicked
+  for (let folder of closedFolders) {
+    await folder.click();
+  }
+};
+
+const checkFocus = async function (t, selector, index) {
```

Comment:
This is defined in two files (or more, I noticed 2). Does it make sense to consolidate these implementations in a followup?

Reply:
I think you are right, this should be broken out.